README.adoc

```diff
@@ -385,6 +385,26 @@ class Transaction < ActiveRecord::Base
 end
 ----
 
+=== Enums [[enums]]
```

Comment:
I investigated whether `=== Enums` or `=== Enum` would be better. The API says `Enum`, but the guide says `Enums`, so I think that `Enums` would be fine.

- Guide ... https://guides.rubyonrails.org/active_record_querying.html#enums
- API ... https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html